tests/screenshot/apps/components/Layout.js

```diff
@@ -76,6 +94,24 @@ const columnTestCases = [
 		<Cell className={css.blue} shrink>Column header5</Cell>
 		<Cell className={css.red}>Column Content5</Cell>
 		<Cell className={css.blue} size={500}>Column footer5</Cell>
+	</Column>,
+
+	<Column style={{height: "500px"}}>
+		<Cell className={css.red} grow>Column header6</Cell>
+		<Cell className={css.blue} grow>Column Content6</Cell>
+		<Cell className={css.green} grow>Column footer6</Cell>
+	</Column>,
+
+	<Column style={{height: "500px"}}>
+		<Cell className={css.red} grow size={100}>Column header7</Cell>
```

Comment:
Likewise, the previous one and this test look similar.

Reply:
Same as the intent of the test above.